docs/application/web/guides/multimedia/media-controller.md

```diff
@@ -152,6 +156,27 @@ To manage the media controller features in your application, you must learn to h
    mcServerInfo.sendPlaybackState('PAUSE', successCallback, errorCallback);
    ```
 
+## Setting and Getting the Server Icon URI
```

Comment:
To maintain consistency, please avoid articles from heading.
Setting and Getting Server Icon URI